roles/install_yamls/defaults/main.yml

```diff
@@ -35,3 +35,6 @@ cifmw_install_yamls_whitelisted_vars:
   - OUTPUT_BASEDIR
   - OUTPUT_DIR
   - SSH_KEY_FILE
+# Defines in install_yamls when we should clone and checkout based on
+# openstack-operator references.
+cifmw_install_yamls_checkout_openstack_ref: "true"
```

Comment:
Could you please add `cifmw_install_yamls_checkout_openstack_ref` to the role README

Reply:
good catch :) 
I will provide an update, thanks